### PR TITLE
tabs updates without animation change

### PR DESCRIPTION
**Pull request by darintrimillos**
## Description
This branch has all the tabs changes, except for the animation changes, which will get further reviewed by design. I also updated based on @cowills feedback to rename activateTab and simplify the setup in the mounted hook.

Jira Tickets:
CDR-1616
CDR-1617
CDR-1603
CDR-1604

### Design:
- [ ] Reviewed with designer and meets expectations

### Cross-browser testing:
- [x] Chrome
- [x] Firefox
- [x] Edge
- [x] Safari
- [x] IE11
- [ ] iOS
- [ ] Android

### Visual regression testing:
- [x] Added/updated backstop tests
- [ ] Passes with existing reference images (or failed as expected)

### Unit testing:
- [x] Sufficient unit test coverage (see unit test best practices for ideas)

### A11y:
- [x] Meets WCAG AA standards

### Documentation:
- [ ] API docs created/updated
- [ ] Examples created/updated

### CODE REVIEW

File: src/components/tabs/CdrTabs.jsx [hunk 1]
```diff
@@ -44,6 +49,8 @@ export default {
       .map(vnode => vnode.componentInstance)
       .filter(tab => tab); // get vue component children in the slot
 
+    this.activeTabIndex = this.getNextTab();
```
Comment (cowills) [inline src/components/tabs/CdrTabs.jsx]: `this.activeTabIndex = this.getNextTab(this.activeTab);` for activeTab to work

Comment (darintrimillos) [reply src/components/tabs/CdrTabs.jsx]: Whoops. Fixed.

**Comment by lancetharper** [comment]
I know this is meant to be links, but I wonder if we need to emit a click event too like pagination for people using vue-router and other routing oddities

**Comment by cowills** [comment]
> I know this is meant to be links, but I wonder if we need to emit a click event too like pagination for people using vue-router and other routing oddities

i think we could solve that in a generic way by giving full programmatic control over which tab is active...though i can't really think of a case where someone would want or need to do that.

**Comment by lancetharper** [comment]
> > I know this is meant to be links, but I wonder if we need to emit a click event too like pagination for people using vue-router and other routing oddities
> 
> i think we could solve that in a generic way by giving full programmatic control over which tab is active...though i can't really think of a case where someone would want or need to do that.

Wouldn't you still have to `preventDefault` the link click though?

**Comment by cowills** [comment]
> Wouldn't you still have to `preventDefault` the link click though?

yeah, i guess my point is if we want to support that there's other work to do besides exposing the event, and i can't think of a use case for it either

**Review by cowills — DISMISSED** [review]

**Review by cowills — APPROVED** [review]

**Pull request merged**